prisma/migrations/20210707085343_releases/migration.sql

```diff
@@ -1,6 +1,6 @@
 -- CreateTable
```

Comment:
We should not move / edit migrations. If the previous migration was applied already, this will break the automated deploy since the table would already exist. Instead create a new migration to alter the table appropriately.